.github/labeler.yml

```diff
@@ -176,3 +176,12 @@
 'category: licensing':
 - 'licensing/**/*'
 - 'LICENSE'
+
+'category: docker_env':
+- '.dockerignore'
+- 'install_build_dependencies.sh'
+- '**/install_openvino_dependencies.sh'
+- '.github/actions/handle_docker/**/*'
+
+'category: dockerfiles':
```

Comment:
looks strange that label are actually named without `category` 
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/139c3e6d-5616-419d-9f0b-80bea6c59bb8)

Reply:
Those are placed manually (labeler configuration is applied only after merge)

Reply:
can we remove these labels then>

Reply:
I raised a separate PR with labeler configuration: https://github.com/openvinotoolkit/openvino/pull/24309
Once it's merged, I'll remove manually created labels